heft-plugins/heft-jest-plugin/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@rushstack/heft-jest-plugin",
+  "version": "0.1.0",
+  "description": "Heft plugin for Jest",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/rushstack/tree/master/heft-plugins/heft-jest-plugin"
+  },
+  "homepage": "https://rushstack.io/pages/heft/overview/",
+  "main": "lib/index.js",
+  "types": "dist/heft-jest-plugin.d.ts",
+  "license": "MIT",
+  "scripts": {
+    "build": "heft test --clean",
+    "start": "heft test --clean --watch"
+  },
+  "peerDependencies": {
+    "@rushstack/heft": "^0.32.0"
```

Comment:
This version has not been published yet.

Shouldn't the peer dependency be like this?
```
  "@rushstack/heft": "workspace:^0.31.4"
```

Reply:
This is intentional. The existing version of Heft doesn't have support for the plug-in, but will once a publish run is performed.

Reply:
Oh, it it's a workaround for a `cyclicDependencyProjects` relationship?

Reply:
Yes it is unfortunately. We want to consume the plugin in the rig, but also want the rig to build our plugin. Catch-22